chaoslib/litmus/cordon_drain_node.yaml

```diff
@@ -0,0 +1,45 @@
+- block:
+
+  - name: Identify the application node
+    shell: >
+      kubectl get pod {{ app_pod }} -n {{ app_ns }} 
+      --no-headers -o custom-columns=:spec.nodeName
+    args: 
+      executable: /bin/bash
+    register: result 
+             
+  - name: Record the application node name 
+    set_fact:
+      app_node: "{{ result.stdout }}"
+  
+  - name: Drain the application node
+    shell: >
+      kubectl drain {{ app_node }}
+      --ignore-daemonsets --delete-local-data  --force
+    args:
+      executable: /bin/bash
+    register: result
+    until: "'cordoned' in result.stdout"
+    delay: 20
+    retries: 12
+  
+  - name: Wait for application pod reschedule (evict)
```

Comment:
@shubhamchaudhary  Is this Timeout interval is enough to evict the application pod. Can you include a task to check the application pod is evicted. instead of giving wait time.